hw/rtl/VX_types.vh

```diff
@@ -196,4 +196,10 @@
 `define VX_CSR_NUM_CORES                12'hFC2
 `define VX_CSR_LOCAL_MEM_BASE           12'hFC3
 
+`define VX_MAT_MUL_SIZE                 12'hFC4
```

Comment:
What is the relationship between matmul size and tensor core size?

Reply:
Added a comment.